### PR TITLE
Support for PodSecurityPolicies

### LINKED ISSUE #72: Support for PodSecurityPolicies
In order to install the provisioner in clusters which have PodSecurityPolicy in effect, a policy fulfilling the configured security contexts is required

### PR DESCRIPTION
**Pull request by weargoggles**
Add a PodSecurityPolicy, Role and RoleBinding to allow the provisioner's security contexts on clusters where PodSecurityPolicy admission control is in effect.

Fixes #72

### CODE REVIEW

**Comment by weargoggles** [comment]
I signed it

**Comment by cofyc** [comment]
You can run `./hack/update-generated.sh ` to update all generated example files.

File: helm/provisioner/templates/provisioner-cluster-role-binding.yaml [hunk 1]
```diff
@@ -82,4 +82,41 @@ roleRef:
...
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: {{ replace "+" "_" .Chart.Version | printf "%s-%s" .Chart.Name }}
+rules:
+- apiGroups:
+    - policy
+  resources:
+    - podsecuritypolicies
+  verbs:
+    - use
```
Comment (cofyc) [inline helm/provisioner/templates/provisioner-cluster-role-binding.yaml]: should we use `resourceNames` to authorize `local-storage-provisioner-pod-security-policy` policy only?

**Pull request merged**